securing_apps/topics/oidc/java/spring-security-adapter.adoc

```diff
@@ -6,6 +6,14 @@ You then have to provide some extra beans in your Spring Security configuration
 
 Unlike the other Keycloak Adapters, you should not configure your security in web.xml.
 However, keycloak.json is still required.
+In order for Single Sign Out work properly you have to define a session listener in web.xml:
+[source,xml]
+---- 
+<listener>
+     <listener-class>org.springframework.security.web.session.HttpSessionEventPublisher</listener-class>
+</listener>
+----
+
```

Comment:
Maybe add the https://stackoverflow.com/questions/24955534/spring-3-5-how-to-add-httpsessioneventpublisher-to-my-boot-configuration/24957247#24957247 possible way ?

Reply:
ok, I'll add it! @sebastienblanc

Reply:
> Maybe add the https://stackoverflow.com/questions/24955534/spring-3-5-how-to-add-httpsessioneventpublisher-to-my-boot-configuration/24957247#24957247 possible way ?

Done @sebastienblanc